elastic_transport/_exceptions.py

```diff
@@ -50,9 +50,6 @@ def __repr__(self) -> str:
             parts.append(f"errors={self.errors!r}")
         return "{}({})".format(self.__class__.__name__, ", ".join(parts))
 
-    def __str__(self) -> str:
-        return str(self.message)
-
```

Comment:
Since `message` is passed in `super.__init__(message)` and we removed the custom `__str__` implementation below, this is already the default implementation.

In Python, `__str__` isn't helpful. If you need to know the class name, then `__repr__` is required, and our `__repr__` implementation contains all information about the current exception, including `errors`.

```python
>>> e = ValueError("abandon ship!")
>>> str(e)
'abandon ship!'
>>> repr(e)
"ValueError('abandon ship!')"
```